### DISCUSSION TITLE
Removal of CE features: Can you still trust RC as a long-term solution?

### DISCUSSION THREAD

**Discussion by MarcoSteinacher** [RocketChat/Rocket.Chat#33870] [General]
Sad to hear that private apps won't be allowed anymore in Rocket.Chat from version 7.0. With this, Rocket.Chat AGAIN removes an existing feature from the Community Edition and moves it to the Enterprise Edition. This makes me think about how trustworthy Rocket.Chat is as a long-term solution. If we have to fear that with every release a feature we depend on might be removed, maybe we should move on to a «real» open-source solution.

I understand and support the concept of having a Community and an Enterprise edition. But removing existing features from the CE is a NO-GO for this model to work. It gives the impression that «open source» and «community» become merely a marketing terms and not values that are really lived by.

I know that existing private apps should continue to work in CE. But still, the feature of adding private apps IS removed from CE. Also, what if we have to update or extend our existing private app? I think this will not be possible anymore.

On the website this change is explained as «...designed to maintain the integrity of Rocket.Chat's customization options [...] protecting both the value and security of our platform.». What does that even mean? I think this mainly marketing blah-blah and it means, that Rocket.Chat is not afraid of crippling the CE to try to push people to a paid plan. This, in turn, makes me think twice if Rocket.Chat is the right long-term solution for us.

**Comment by Ulire** [comment]
I believe make a private fork and pay for ios developer account (to maintain the version by yourself) might is more cheaper (only 99 USD each year) and stable......

**Comment by reetp** [comment]
Something that has been discussed at length here already is about what open source really means.

Rocket.Chat is open source. The code is there for you to take and fork and build as you see fit. Don't like something? Fork, remove/modify, and build it yourself. Nope you can't use the EE code, but the rest is fair game.

But if you want to use a pre-built binary then it is on their terms.

Somewhere along the way they have to make a profit to pay the devs to build the software.

Remember, if they go bust you get _nothing at all._ They have tried donations and various methods and none of them make much of a dent. When asked who should pay, users wag fingers and say "those people over there should pay". It is never them.

So who should pay?

They offer a fully functioning system for up to 50 users for free. That is a pretty good deal.

If you have more users than that and are making money on the back of their work then why aren't you contributing in some way?

I don't see people round here doing what I do - helping with bugs, or support, or writing docs - they pop in occasionally when it suits and they need help. 

Community is also about giving back, not just taking all the time. "I'm too busy". Yup, we all are. I have a business to run as well. And?

If you are a NFP or other type of charitable institution (remember, even charities have donations and expenses) then talk to sales and you will get a sympathetic ear.

Even if you are a business then they will be happy to negotiate.

As far as I am aware (I am not an employee, but did work for Rocket for a while and have many good friends there) there is no intention to move anything else to EE.

However, I am not an employee, and nothing is set in stone so I cannot guarantee that.

I believe they are goig to start running a community call again shortly - I may even be on it! - and hopefully we can get some answers to allay your fears.

**Reply by MarcoSteinacher** [reply]
I agree with you in most points. I understand that devs need to get paid (I'm a dev myself) and I'm not saying others should pay or that the Community is not about giving back. I'm not challenging the dual model with CE and EE here either. My only concern is that great open-source projects can be crippled over time because functionality is removed or moved to the EE for profit reasons. Of course Rocket.Chat is free to make those decisions. I just wanted to express that this is not what I expect from such a project and that I believe that it damages trust. That's all.

**Reply by reetp** [reply]
>  I just wanted to express that this is not what I expect from such a project and that I believe that it damages trust.

What do you expect?

What do expect from other projects?

What would make Rocket different specifically?

Curious as I've heard similar many times, and the criticism is only directed at software people use.

See my comment above about finger pointing.

Who do you think should pay?

**Comment by casalsgh** [comment]
Hi MarcoSteinacher, 
Private apps will still be allowed if you register the workspace and have up to 50 active seats. That is the Starter plan.
Even if local you can still register and get the air-gapped feature. You can check more info at https://www.rocket.chat/pricing 

Notice that the type of movement being performed here is exactly to place Rocket.Chat as a long-term solution that folks can trust will still be in the market 20 years from now and much beyond that. 
Hope this information on the Starter plan helps and thanks for bringing up your concerns. 

Rgds
